_glaredb/integrations/hyperquery.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Hyperquery
+parent: Integrations
+---
+
+# Hyperquery
+
+[Hyperquery] is a data notebook built for speed, visibility, and collaboration.
+You can write and execute SQL queries or Python cells directly within a WYSIWYG
+(what you see is what you get) notebook, and augment this work with rich
+formatting options.
+
+In this guide, we'll walk through how to get starting with GlareDB in
+Hyperquery.
```

Comment:
delete this: 
- I'm (also) somewhat opposed to the "we," first person. It's kind of folksy, and approachable which might be fun, but I think there's some benefit of the polish formality.
- I think it would be good if the title of document has something like "Using GlareDB in Hyperquery" as the title or something (for search engine reasons,) 
- sometimes it's good as a sort of "tl;dr" for things like this to state the final outcome or objectives of the document.